types/rdfjs__fetch-lite/Factory.d.ts

```diff
@@ -11,7 +13,11 @@ interface RdfFetchResponse<
 }
 
 interface Fetch {
-    (url: Parameters<typeof fetch>[0], options?: FormatsInit): Promise<RdfFetchResponse<DatasetCore>>;
+    // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
```

Comment:
Is there a way of avoiding this disabling

Reply:
I don't know